cognite_toolkit/_cdf_tk/commands/featureflag.py

```diff
@@ -61,7 +62,7 @@ def is_enabled(flag: str | Flags) -> bool:
     @lru_cache
     def to_flag(flag: str) -> Flags | None:
         try:
-            return Flags[flag.upper()]
+            return Flags[flag.upper().replace("-", "_")]
```

Comment:
It feels more natural for met to write hyphen in a terminal than `_`, so I think I am not the only one this applies to.
```
cdf-tk features set import-cmd --enable`
```

Reply:
Yes, me too 😆